test/skills.e2e-spec.ts

```diff
@@ -0,0 +1,160 @@
+import { INestApplication } from '@nestjs/common';
+import { Test } from '@nestjs/testing';
+import { DataSource } from 'typeorm';
+import * as request from 'supertest';
+import type { App } from 'supertest/types';
+import { SkillsService } from '@/skills/skills.service';
+import { CategoriesService } from '@/categories/categories.service';
+import { AppModule } from '@/app.module';
+import { User } from '@/entities/user.entity';
+import { Skill } from '@/entities/skill.entity';
+import { CreateSkillDto } from '@/skills/dto/create-skill.dto';
+import { FindSkillsQueryDto } from '@/skills/dto/find-skills.dto';
+import { UpdateSkillDto } from '@/skills/dto/update-skill.dto';
+import { UUID } from 'crypto';
+import { Category } from '@/categories/entities/category.entity';
+import { UsersService } from '@/users/users.service';
+
+describe('Skills тесты (e2e)', () => {
+  let app: INestApplication;
+  let dataSource: DataSource;
+  let skillsService: SkillsService;
+  let createdSkill: Skill;
+  let categoryService: CategoriesService;
+  let userService: UsersService;
+  let userId: UUID;
+  let categories: Category[];
+  let category: Category;
+  let server: any;
+  let loginResponse: any;
+  let token: any;
+  let user: User;
+  let userEmail: string;
+  let userPassword: string;
+
+  beforeAll(async () => {
+    const module = await Test.createTestingModule({
+      imports: [AppModule],
+    }).compile();
+
+    app = module.createNestApplication();
+    await app.init();
+
+    dataSource = app.get(DataSource);
+    await dataSource.runMigrations();
+
+    skillsService = app.get(SkillsService);
+    const { data } = await skillsService.findAll({ page: 1, limit: 20 });
+
+    userId = data[0].owner.id;
```

Comment:
Изначально у нас в бд нет навыков, поэтому Id пользователя нужно взять при входе
```suggestion
    userId = loginResponse.body.user.id
```